advanced-usage/index.md

```diff
@@ -83,26 +108,24 @@ Choose between:
    - CPU overheating
    - CPU throttling
    - CPU under voltage
+   - Connected wirelessly (instead of through a tether)
 {{ easy_image(src="system-status", width=200) }}
 
 #### Sidebar
 
-The burger menu at the top left of the header opens up the side-bar, for
-conveniently accessing available pages, tools, and services.
+The burger menu at the top left of the header opens up the sidebar, for
+conveniently accessing available pages, tools, and services. When the page
+is wide enough, the sidebar automatically stays open.
 
 {{ simple_pirate_image(src="sidebar", width=200) }}
 
-##### Settings
+##### BlueOS Settings
```

Comment:
There's a whole section of the sidebar currently called "Settings", and there's a heading for that lower down - I thought it made sense for them to be differentiated, and qualified this one as specifically settings for BlueOS (since the other one also includes autopilot settings). This can be changed to something else if/when the sidebar names change (again).